app/models/odk/response_parser.rb

```diff
@@ -15,7 +15,7 @@ def self.duplicate?(xml, user_id)
         attachment = ActiveStorage::Attachment.find_by(blob_id: blob.id)
         next if attachment.nil?
         response = Response.find(attachment.record_id)
-        return true if response.&user.&id == user_id
+        return true if response.present? && response.user.id == user_id
```

Comment:
Ah this was just backwards fyi, should be `&.` instead of `.&`

If helpful for memory jogging, here's the rationale behind this horrible syntax because it doesn't look like most other languages: https://stackoverflow.com/a/36813420/763231 ... you can think of it similar to `response && .user && .id` where it doesn't continue unless the preceding thing exists.

Fine to leave as-is!